spec/README.md

```diff
@@ -1,3 +1,102 @@
+# Testing styleguide
+
+We use ["Futurelearn style" acceptance tests](https://about.futurelearn.com/blog/how-we-write-readable-feature-tests-with-rspec). This style was originally from [the "apply" team's spec docs](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/ba9125e9aa5f212c18b63c7c3d49f8785486df83/docs/testing-styleguide.md)
```

Comment:
I think it should be clear that this is talking about feature tests, or are we proposing that unit and request tests follow these rules too?